.github/workflows/release.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
```

Comment:
NOTE: We had to add `fetch-depth: 0` here for goreleaser to properly create the changelog. See https://github.com/stormforger/cli/blob/master/.github/workflows/release.yml#L20-L23